docs/source/development/traffic_ops_api/v12/profile.rst

```diff
@@ -71,6 +73,7 @@ Profiles
             "cdn": 1
             "cdnName": "cdn1"
             "type": "ATS_PROFILE"
+            "routingDisabled": 0
```

Comment:
booleans should be returned or submitted as true|false not 1|0. this changed when we migrated to postgres.

Reply:
That's what they're coming out of the db function as - they are in the DB as booleans.  Should we manually change them to true|false at the json level?

Reply:
look at deliveryservice.pm...lots of booleans coming out of that api